base/shell/explorer/startmnucust.cpp

```diff
@@ -203,36 +227,42 @@ INT_PTR CALLBACK CustomizeClassicProc(HWND hwnd, UINT Message, WPARAM wParam, LP
         case WM_INITDIALOG:
             CustomizeClassic_OnInitDialog(hwnd);
             return TRUE;
+
         case WM_COMMAND:
             switch (LOWORD(wParam))
             {
                 case IDC_CLASSICSTART_ADD:
                     OnAddStartMenuItems(hwnd);
                     break;
+
                 case IDC_CLASSICSTART_REMOVE:
                     OnRemoveStartmenuItems(hwnd);
                     break;
+
                 case IDC_CLASSICSTART_ADVANCED:
                     OnAdvancedStartMenuItems();
                     break;
+
                 case IDC_CLASSICSTART_CLEAR:
                     OnClearRecentItems(hwnd);
                     break;
+
                 case IDOK:
                     if (CustomizeClassic_OnOK(hwnd))
-                    {
                         EndDialog(hwnd, IDOK);
-                    }
                     break;
+
                 case IDCANCEL:
                     EndDialog(hwnd, IDCANCEL);
                     break;
             }
             break;
+
```

Comment:
I would strip all those added newlines between the case-statements. Having vertical compact switch statements is the greater good imho than visually separating the individual cases.

Reply:
Maybe for that switch inside the case WM_COMMAND, but otherwise, having the newlines for the cases of the outer switch statement have been added for readability.

Reply:
Deleted new lines.